doc/manuals/orion-api.md

```diff
@@ -1134,6 +1105,42 @@ The values that `options` parameter can have for this specific request are:
 | `keyValues` | when used, the response payload uses the `keyValues` simplified entity representation. See "Simplified Entity Representation" section for details. |
 | `upsert`    | when used, entity is updated if already exits. If upsert is not used and the entity already exist a `422 Unprocessable Entity` error is returned.  |
 
+**Request payload**
+
+Content-Type is `application/json`
+
+Example:
+
+```json
+{
+  "type": "Room",
+  "id": "Bcn-Welt",
+  "temperature": {
+    "value": 21.7
+  },
+  "humidity": {
+    "value": 60
+  },
+  "location": {
+    "value": "41.3763726, 2.1864475",
+    "type": "geo:point",
+    "metadata": {
+      "crs": {
+        "value": "WGS84"
+      }
+    }
+  }
+}
+```
+
+**Response**
+
+* Successful operation uses 201 Created (if upsert option is not used) or 204 No Content (if
+  upsert option is used). Response includes a `Location` header with the URL of the
+  created entity.
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
+  more details.
+
 + Response 201
 
     + Headers
```

Comment:
Should we leave this part like this? (examples of responses with just the header)